asciidoc/courses/app-go/modules/2-interacting/lessons/1-transactions/lesson.adoc

```diff
@@ -91,7 +91,7 @@ For this reason, these should only be used for one-off queries and shouldn't be
 When you intend to read data from Neo4j, you should execute a **Read** Transaction.
 In a clustered environment (including Neo4j Aura), read queries are distributed across the cluster.
 
-The session provides a `ReadTransaction()` function, which expects a single parameter, a function that represents the unit of work.
+The session provides an `ExecuteRead()` function, which expects a single parameter, a function that represents the unit of work.
```

Comment:
```suggestion
The session provides an `ExecuteRead()` function, which expects at least two parameters, a `context.Context` and a function that represents the unit of work.
```

Reply:
This feels like the issue has been glossed over a bit.  Do we need to specify what the context is or where/how one is created?